### ISSUE TITLE
how to set padding of new modal sheet from top?

### ISSUE THREAD

**Issue by jinfagang** [jamesblasco/modal_bottom_sheet#61]
![image](https://user-images.githubusercontent.com/21303438/90782220-6a077e00-e331-11ea-859d-3f2fc5e22cd7.png)

like this, the bottom sheet have a larger padding , how to make it looks like this?

**Comment by jamesblasco** [comment]
Not sure how is the interaction between those multiple modals.

But it is not possible to change the top padding of the showCupertinoModalBottonSheet.

You can create your own modal bottom sheet similar to https://github.com/jamesblasco/modal_bottom_sheet/blob/master/example/lib/modals/floating_modal.dart. 
But this does not animate previous routes

**Comment by bierbaumtim** [comment]
I currently working on stacking modals because it is not working as expected. 
I looked into the iOS Calendar App and stacked modals have the same padding but they have a different closing behavior. If you close the stacked modal the previous modal starts the transition when you reach around the half of the screen. 
When you have one modal and closes it, the previous route starts the transition immediately.

I thinks in the example are one modal and a second one which uses a different presentationStyle without the modal transition.

**Comment by tomasdev** [comment]
Shouldn't this be easy to solve as long as `_behind_widget_visible_height` becomes a parameter?

**Comment by jinfagang** [comment]
@tomasdev Yeah, I agree with tomas, any concern on why not make it as a param?

**Comment by ekoRemDev** [comment]
url is not valid 

there is updated i believe
https://github.com/jamesblasco/modal_bottom_sheet/blob/main/modal_bottom_sheet/example/lib/modals/floating_modal.dart
